<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>371</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>          Serial Port | Parameters</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Baudrate :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbBaudRate">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Data bits : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Parity : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop Bits : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Flow Control : </string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbDataBits">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cmbParity">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cmbStopBits">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cmbFlowControl">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Port List :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbPortList">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>240</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>COM13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM11</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnConnection">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>280</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>281</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QPushButton" name="btnVref">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>btnVref</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTemp">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>btnTemp</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnButtonState">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>btnButtonState</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAll">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>btnAll</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblOutput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>161</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>lblOutput</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
